app/javascript/components/App.js

```diff
@@ -50,12 +92,37 @@ export default class App extends Component {
             <Route
               path="/patientinfo/:id"
               render={(props) => {
-                let id = +props.match.params.id;
-                console.log(id);
-                let patientInfo = this.state.patientsArray.find(
+                const id = +props.match.params.id;
```

Comment:
is this a typo? `+props`

Reply:
@archae0pteryx  the `+` is how we were taught as a shorthand to turn a `String` into a `Number`

Reply:
Ah yes. Being a url, i wonder if it needs to be coerced?

Reply:
Ah good question, I never questioned it cuz that's the example we followed but I can take it off and see

Reply:
@archae0pteryx i did a `console.log(typeof props.match.params.id);` and it outputted `string` so it looks like it does need to be coerced